test/pytest/test_pytorch_profiler.py

```diff
@@ -137,17 +138,22 @@ def test_batch_input(set_custom_handler, handler_name):
     """
     Tests pytorch profiler integration with batch inference
     """
+
     CUSTOM_PATH = "/tmp/output/resnet-152-batch"
+
     if os.path.exists(CUSTOM_PATH):
         shutil.rmtree(CUSTOM_PATH)
     assert os.path.exists(data_file_resnet)
 
-    cmd = ["bash", os.path.join(profiler_utils, "resnet_batch.sh")]
+    def invoke_batch_input():
+        data = open(data_file_resnet, "rb")
+        response = requests.post("{}/predictions/resnet152".format(TF_INFERENCE_API), data)
+        print(response.text)
 
-    proc = subprocess.run(cmd, stdout=subprocess.PIPE, check=True, timeout=1000)
 
-    assert "tiger_cat" in proc.stdout.decode("utf-8")
-    assert "Labrador_retriever" in proc.stdout.decode("utf-8")
+    with futures.ThreadPoolExecutor(2) as executor:
+        for _ in range(2):
+            executor.submit(invoke_batch_input)
```

Comment:
1. The asserts were deleted. Where is the verification of the results? 
2. I don't fully understand why threadpooling can fix regression test? Could you pls elaborate the original issue?

Reply:
The objective of the test case (test_batch_input) : When user provides inputs as a batch, only 1 PyTorch trace file should be generated. 

Initially i created the test case with a shell script - the shell script invokes 2 curl commands. The regression test is failing at this line - while forming the path for the shell script.
https://github.com/pytorch/serve/blob/b11ba0fa97ceea7003bfa7c4fd4203f36fee3d34/test/pytest/test_pytorch_profiler.py#L145 .

Instead of fixing the path construction, i thought it would be more simpler to use python threads. [This line](https://github.com/pytorch/serve/blob/b11ba0fa97ceea7003bfa7c4fd4203f36fee3d34/test/pytest/test_pytorch_profiler.py#L152) verifies the test case.